docs/modules/development/pages/rest/snmp_configuration.adoc

```diff
@@ -75,34 +75,34 @@ All parameters can be set regardless the version.
 .Example 1:
 [source, bash]
 ----
-curl -v -X PUT -H "Content-Type: application/xml" \
+curl -v -u admin:admin -X PUT -H "Content-Type: application/xml" \
      -H "Accept: application/xml" \
-     -d "&lt;snmp-info&gt;
-             &lt;community&gt;yRuSonoZ&lt;/community&gt;
-             &lt;port&gt;161&lt;/port&gt;
-             &lt;retries&gt;1&lt;/retries&gt;
-             &lt;timeout&gt;2000&lt;/timeout&gt;
-             &lt;version&gt;v2c&lt;/version&gt;
-          &lt;/snmp-info&gt;" \
-     -u admin:admin http://localhost:8980/opennms/rest/snmpConfig/10.1.1.1
+     -d "<snmp-info>
+             <community>yRuSonoZ</community>
+             <port>161</port>
+             <retries>1</retries>
+             <timeout>2000</timeout>
+             <version>v2c</version>
+          </snmp-info>" \
+     http://localhost:8980/opennms/rest/snmpConfig/10.1.1.1
 ----
 
 Creates or updates a `<definition/>`-entry for IP address 10.1.1.1 in `snmp-config.xml`.
 
 .Example 2:
 [source, bash]
 ----
-curl -v -X GET -u admin:admin http://localhost:8980/opennms/rest/snmpConfig/10.1.1.1
+curl -v -u admin:admin -X GET ://localhost:8980/opennms/rest/snmpConfig/10.1.1.1
 ----
 
 Returns the SNMP configuration for IP address 10.1.1.1 as defined in example 1.
 
-== API Version 2
+== API version 2
 
 Since version 2, all attributes of a `<definition />` entry defined in `snmp-config.xsd` (http://xmlns.opennms.org/xsd/config/snmp) can be set or get via the interface - except it is only possible to set the configuration for one IP address and not for a range of IP addresses.
```

Comment:
```suggestion
Since version 2, you can set or get all attributes of a `<definition />` entry defined in `snmp-config.xsd` (http://xmlns.opennms.org/xsd/config/snmp) via the interface, except it is only possible to set the configuration for one IP address and not for a range of IP addresses.
```

Reply:
```suggestion
Since version 2, all attributes of a `<definition />` entry defined in `snmp-config.xsd` (http://xmlns.opennms.org/xsd/config/snmp) can be set or get via the interface, however it is only possible to set the configuration for one IP address and not for a range of IP addresses.
```